src/main/java/ladder/domain/Row.java

```diff
@@ -14,30 +17,45 @@ public Row(List<Boolean> cols) {
     }
 
     private void validate() {
-        for (int i = 0; i < cols.size() - 1; i++) {
+        validateSize();
+        validateDuplicate();
+    }
+
+    private void validateSize() {
+        if(this.cols.size() < MIN_COUNT_OF_COLS) {
+            throw new InvalidColSizeException();
+        }
+    }
+
+    private void validateDuplicate() {
+        for (int i = 0; i < this.cols.size() - 1; i++) {
```

Comment:
```suggestion
        for (int i = 0; i < this.size(); i++) {
```

저는 해당 로직을 메소드로 분리시킬것같아요